theano/tensor/tests/test_opt.py

```diff
@@ -4100,6 +4100,8 @@ def test_local_mul_switch_sink(self):
                             self.resm[idx][i])).sum() == self.resm[idx][i].size
                 else:
                     res = f(condition[1], x[1], -1)
+                    theano.printing.debugprint(f.maker.fgraph.outputs[0])
```

Comment:
It seem this line come back. Can you remove it? It make the tests extra verbose and this isn't good.